x-pack/plugins/fleet/server/services/epm/elasticsearch/template/default_settings.test.ts

```diff
@@ -21,45 +18,10 @@ mockedAppContextService.getSecuritySetup.mockImplementation(() => ({
   ...securityMock.createSetup(),
 }));
 
-let mockedLogger: jest.Mocked<Logger>;
 describe('buildDefaultSettings', () => {
-  beforeEach(() => {
-    mockedLogger = loggerMock.create();
-    mockedAppContextService.getLogger.mockReturnValue(mockedLogger);
-  });
-
-  it('should generate default settings', () => {
+  it('should not generate default settings', () => {
```

Comment:
nit - it's still generating default settings though, right? Just no `default_field` setting anymore